code/modules/overmap/exoplanets/decor/turfs/water.dm

```diff
@@ -51,3 +138,128 @@
 	name = "putrid sewage"
 	desc = "This is utterly vile."
 	color = "#9ea844"//Ew
+
+/turf/simulated/floor/exoplanet/water/shallow/sewage/process()
+	. = ..()
+	reagents.add_reagent(/singleton/reagent/toxin, 2)
+	for(var/mob/living/carbon/human/H in src)
+		// add toxins
+		H.reagents.add_reagent(/singleton/reagent/ammonia, rand(10,30))
+		H.reagents.add_reagent(/singleton/reagent/acid/hydrochloric, rand(10,30))
+		H.reagents.add_reagent(/singleton/reagent/toxin, rand(10,30))
```

Comment:
this is going to give people a billion of these reagents if they stay for too long, you should probably add a reagent amount check where it doesn't add anything if it's over 30-40